chaos.py

```diff
@@ -0,0 +1,27 @@
+#!/bin/python3
+
+import sys
+
+def minimumBribes(q):
+    # Complete this function
+    bribes = len(q)
+    pos = {1,2}

```

Comment:
I am not familiar with this algorithm. It seems to work on simple input but I have my doubt whether it works in general.

In particular, this initialization starts with 2 and we add the following element in the loop but it looks pretty artificial (why wouldn't starting with just the first element be enough?).

The algorithm would need some explanations, including any existing algorithm you're adapting to the problem.